libcore/ProgressInfo.vala

```diff
@@ -17,25 +17,22 @@
  */
 
 public class PF.Progress.Info : GLib.Object {
-    public signal void changed ();
-    public signal void progress_changed ();
-    public signal void started ();
-    public signal void finished ();
-
     private const uint SIGNAL_DELAY_MSEC = 100;
 
-    private GLib.Cancellable cancellable;
-
-    private string title;
-    private string status;
-    private string details;
-    private double progress;
-    private double current;
-    private double total;
-    private bool activity_mode;
-    private bool is_started;
-    private bool is_finished;
-    private bool is_paused;
+    public double progress { get; private set; default = 0; }
+    public double current { get; private set; default = -1; }
+    public double total { get; private set; default = -1; }
+    public bool activity_mode { get; private set; default = true; }
+
+    public uint inhibit_cookie { get; set; default = -1; }
```

Comment:
```suggestion
    public uint inhibit_cookie { get; set; default = 0; }
```
It's an uint so -1 gives uint.MAX technically. Also, the documentation on inhibit stands that it returns:
> A non-zero cookie that is used to uniquely identify this request.
So the code should be changed accordingly (meaning that >=0 should be != 0)

Reply:
Thanks.  I was just porting the C code but you are right, if a valid cookie is always non-zero then it is not necessary to default to -1.